java/ql/src/experimental/Security/CWE/CWE-691/InsecureClassLoading.qll

```diff
@@ -0,0 +1,49 @@
+import java
+import semmle.code.java.dataflow.FlowSources
+import semmle.code.java.dataflow.TaintTracking2
+
+/**
+ * A taint-tracking configuration for insecure context creation vulnerabilities
+ */
+class InsecureContextCreationConfig extends TaintTracking::Configuration {
```

Comment:
Configurations should in general be defined only in query files (`.ql`), so this should be moved into `InsecureClassLoading.ql`.

Reply:
If you want, you can simply move all of this file into the query file.